### PR TITLE
4683 Installing webpack

### PR DESCRIPTION
**Pull request by AnfFun**

### CODE REVIEW

**Comment by AnfFun** [comment]
@Y0r

**Comment by AnfFun** [comment]
**ISSUE** - https://redmine.devbranch.work/issues/4683?issue_count=3&issue_position=1&next_issue_id=4673

**Comment by AnfFun** [comment]
My bad about commit, always forgot about number of Issue in commit

File: web/themes/custom/newspaper11/newspaper11.libraries.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 global-styling:
   version: 1.x
   css:
+    component:
+      dist/style.css: {}
```
Comment (Y0r) [inline web/themes/custom/newspaper11/newspaper11.libraries.yml:5]: Why do you need to attach the same file on different levels? 
Just leave theme level, and remove component level.

**Review by Y0r — APPROVED** [review]
Solve issue with theme libraries file and push it into targe branch.

**Pull request merged**